### DISCUSSION TITLE
How to use themes from CDN?

### DISCUSSION THREAD

**Discussion by elmonty** [atomiks/tippyjs#1134] [Q&A]
I am linking Tippy from the CDN. I am trying to use the light theme but simply adding "theme:'light'" to the props doesn't work.

What am I missing?

**Comment by alfredgg** [comment]
Like this:
`<link rel="stylesheet" href="https://unpkg.com/tippy.js@6/themes/light.css" />`
https://tippyjs.bootcss.com/getting-started/#node-vs-browser
